### PR TITLE
Fix Init issues

### PR DESCRIPTION
**Pull request by sidharthv96**
Some issues I faced when initializing a new project.

Also had to do `yarn add -D @types/inquirer @types/fs-extra`. But not sure where to add them in here.

### CODE REVIEW

File: packages/cli-plugin-mongoose/templates/config.hbs [hunk 1]
```diff
@@ -1,8 +1,5 @@
 export default {
   id: "{{name}}",
   url: process.env.{{symbolName}}_URL || "mongodb://localhost:27017/{{name}}",
```
Comment (sidharthv96) [inline packages/cli-plugin-mongoose/templates/config.hbs:3]: Also, this is populated as `DEFAULT_URL` which doesn't seem like the best name for env var for mongo. 
`MONGO_URL` or `DEFAULT_MONGO_URL` might be better?

File: packages/cli-plugin-mongoose/templates/config.hbs [hunk 2]
```diff
@@ -1,8 +1,5 @@
...
-  connectionOptions: {
-    useNewUrlParser: true,
-    useUnifiedTopology: true
-  }
+  connectionOptions: { }
```
Comment (sidharthv96) [inline packages/cli-plugin-mongoose/templates/config.hbs:4]: https://mongoosejs.com/docs/migrating_to_6.html#no-more-deprecation-warning-options

File: packages/cli/templates/init/Dockerfile.hbs [hunk 3]
```diff
@@ -10,22 +10,23 @@
...
 COPY yarn.lock .
+
+RUN yarn install --production
+
 COPY ./src ./src
 COPY ./dist ./dist{{#or swagger oidc}}
 COPY ./views ./views{{/or}}
 
-RUN yarn install --production
-
```
Comment (sidharthv96) [inline packages/cli/templates/init/Dockerfile.hbs:28]: Ensures that code change does not trigger dependency installation on each build.

**Comment by sidharthv96** [comment]
I've been working with Ts.ED for a few months now. But that was an existing project which I progressively migrated.
I wanted to start a new project from start and tested the cli to build it. There were few hiccups that I faced. 
As I'm familiar with the project and also docker and stuff, it was only a minor issue. But the errors thrown could be confusing for a beginner looking to try it out. 

If `tsed init` and `docker compose up` could bring up a working application, that would be ideal.

Few other issues I faced

- `docker compose up` 
  - Won't work on a freshly initialized project which has Mongoose (URL issue)
  - Won't build the project (have to run yarn build manually)
```
$ babel-watch --extensions .ts src/index.ts
alerts-server-1   | TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /src/index.ts
alerts-server-1   |     at new NodeError (internal/errors.js:322:7)
```

I'll try to add other issues that I have to fix to get it to boot here. Hopefully these will help increase adoption of this kickass framework! 🚀

File: packages/cli/templates/init/Dockerfile.hbs [hunk 4]
```diff
@@ -10,22 +10,23 @@
...
 ## date            : 2021-04-14                                              ##
-## version         : 1.1                                                     ##
+## version         : 1.2                                                     ##
 ##                                                                           ##
 ###############################################################################
 ###############################################################################
 FROM node:14-alpine
 
-RUN apk update && apk add build-base git python
+RUN apk update && apk add build-base git python3
```
Comment (sidharthv96) [inline packages/cli/templates/init/Dockerfile.hbs:19]: https://stackoverflow.com/questions/69860233/cant-install-python-package-on-alpine-docker-anymore

**Comment by sidharthv96** [comment]
https://github.com/tsedio/tsed/issues/851

File: packages/cli-plugin-mongoose/templates/config.hbs (hunk 1 above)
Comment (Romakita) [reply packages/cli-plugin-mongoose/templates/config.hbs:3]: it depend. If you have multiple mongoose connection, the env variable must be different per connection.

**Comment by Romakita** [comment]
> Some issues I faced when initializing a new project.
> Also had to do yarn add -D @types/inquirer @types/fs-extra. But not sure where to add them in here.


Probably on  cli-core package.json ;)

File: packages/cli-plugin-mongoose/templates/config.hbs (hunk 1 above)
Comment (sidharthv96) [reply packages/cli-plugin-mongoose/templates/config.hbs:3]: Yup, `{{symbolName}}_MONGO_URL` would help with that right?
Or use a different symbol name (`MONGO`) when generating the initial project.
If a user is explicitly generating new configs, that's fine as they are already somewhat familiar with the framework.

Comment (sidharthv96) [reply packages/cli-plugin-mongoose/templates/config.hbs:3]: DEFAULT_URL in env can mean a different number of things.
DEFAULT_MONGO_URL is more specific.

**Comment by sidharthv96** [comment]
```
    "@types/fs-extra": "9.0.6",
    "@types/globby": "9.1.0",
    "@types/inquirer": "7.3.1",
```

But that's already in there.

Adding here would add them to the project, but these packages are not direct dependencies so that shouldn't be needed right?
https://github.dev/tsedio/tsed-cli/blob/6d3c1db4e782356d0d4dcadc41acd71d7181f9e5/packages/cli/src/commands/init/InitCmd.ts#L307

**Review by Romakita — APPROVED** [review]

**Comment by Romakita** [comment]
I think these dependencies is only required if you use the Command feature in your project. Otherwise isn't necessary :)

**Pull request merged**